scripts/update/Updater.php

```diff
@@ -100,6 +100,16 @@ public function update($initialVersion)
         }
         
         $this->skip('1.2.2', '1.6.1');
+
+        if ($this->isVersion('1.6.1')) {
+            $anonymousUserRole = new \core_kernel_classes_Resource(INSTANCE_ROLE_ANONYMOUS);
```

Comment:
Hello @hutnikau , could you just please make use of the ACLProxy in the Update script?
Thanks a lot !